.travis.yml

```diff
@@ -0,0 +1,13 @@
+language : node_js
+node_js :
+    - "8.11.4"
+services:
+  - mongodb
+before_script:
+  - sleep 15  
+addons:
+  apt:
+    config:
+      retries: true
+    ⋮ # rest (including `packages` and `sources`, if applicable
+   
```

Comment:
Where is the script? what is being executed? I'm guessing you need to check if our tests are running, for that `yarn test` script needs to run on travis

Reply:
Also, put the badge on the very top, before the main bulls and bears heading

Reply:
it automatically detects it

Reply:
No matter, write the script. It's not urgent, take your time. Fix the badge issue urgently though